examples/community/index.test.ts

```diff
@@ -0,0 +1,581 @@
+/**
+ * E2E test suite
```

Comment:
Note: consider containerized waku network instead of mocking js-waku or websocket.